modules/apps/portal-search/portal-search-rest-impl/src/main/java/com/liferay/portal/search/rest/internal/resource/v1_0/SuggestionResourceImpl.java

```diff
@@ -174,6 +174,7 @@ private SearchContext _createSearchContext(
 
 		SearchContext searchContext = new SearchContext();
 
+		searchContext.setAttribute("search.empty.search", Boolean.TRUE);
```

Comment:
Can we use SearchContextAttributes constant here, too?

Reply:
Ok